src/plugin.js

```diff
@@ -76,7 +86,8 @@ module.exports = (config = {}) => {
         namespace: 'muter',
         extends: {
             Chat: extension
-        },
+        }
```

Comment:
Why'd the comma get its own line? Tiny syntax nit-pick

Reply:
Was commenting out lines